tests/roles/cinder_adoption/tasks/main.yaml

```diff
@@ -14,6 +14,13 @@
   retries: 60
   delay: 2
 
+- name: Get cinder-api pod name
+  ansible.builtin.shell: |
+    {{ shell_header }}
+    {{ oc_header }}
+    oc get pod -l component=cinder-api -o jsonpath="{.items[*].metadata.name}"
```

Comment:
-1: This is not going to work outside of `standalone/crc` scenario where `replicas: 3`.
In that case, `cinder_api_pod.stdoutoutlines` is made by > 1 element (e.g. `cinder-5aa52-api-0 cinder-5aa52-api-1 cinder-5aa52-api-2`, so I assume it is sufficient to take the `first` of the list.